### DISCUSSION TITLE
Tagging deprecated keys considering namespace

### DISCUSSION THREAD

**Discussion by filippomuscolino** [tolgee/tolgee-platform#2579] [Q&A] [answered]
Hi, I have a question regarding this feature of the CLI: https://tolgee.io/tolgee-cli/extraction/tagging-keys#tagging-deprecated-keys

We have a big project divided in different namespaces (webapp, mobile, ...) and we would like to manage deprecated keys with CLI as in the example.
Is it possible to tell tag filters to consider only a specific namespace? 

Thanks!

**Comment by JanCizmar** [comment, marked as answer]
Hey! Namespaces are not the right way to handle different platforms. You should be using tags for this. 
Namespaces are designed to split keys info different files while exporting. In the other hand, tags enable you to reuse keys the same keys for different platforms.
